salt/modules/ceph.py

```diff
@@ -0,0 +1,654 @@
+import logging
+
+
+log = logging.getLogger(__name__)
+
+__virtualname__ = 'ceph'
+
+try:
+    import ceph_cfg
+    HAS_CEPH_CFG = True
+except ImportError:
+    HAS_CEPH_CFG = False
+
+
+def __virtual__():
+    if HAS_CEPH_CFG is False:
+        return False, 'The %s execution module cannot be loaded: ceph_cfg unavailable.' % (__virtualname__)
+    return __virtualname__
+
+
+def partition_list():
+    '''
+    List partitions by disk
+
+    CLI Example:
+
+	.. code-block:: bash
+
+        salt '*' ceph.partition_list
+    '''
+    return ceph_cfg.partition_list()
+
+
+def partition_list_osd():
+    '''
+    List all OSD data partitions by partition
+
+    CLI Example:
+
+	.. code-block:: bash
+
+        salt '*' ceph.partition_list_osd
+    '''
+    return ceph_cfg.partition_list_osd()
+
+
+def partition_list_journal():
+    '''
+    List all OSD journal partitions by partition
+
+    CLI Example:
+
+	.. code-block:: bash
+
+        salt '*' ceph.partition_list_journal
+    '''
+    return ceph_cfg.partition_list_journal()
+
+
+def osd_discover():
+    """
```

Comment:
Can you change all of the `"""` references to single quotes like you have above?